components/tabs/src/schema.rs

```diff
@@ -34,10 +31,15 @@ const CREATE_META_TABLE_SQL: &str = "
 const CREATE_PENDING_REMOTE_DELETE_TABLE_SQL: &str = "
     CREATE TABLE IF NOT EXISTS pending_remote_tab_closures (
         id                      INTEGER PRIMARY KEY AUTOINCREMENT,
-        client_id               TEXT NOT NULL,
-        url                     TEXT NOT NULL,
-        time_requested_close    INTEGER NOT NULL
+        device_id               TEXT NOT NULL,
+        command                 TEXT NOT NULL, -- probably 'close'
```

Comment:
Or an `INTEGER` enum field, in the vein of `BookmarkType`, `SyncedBookmarkKind`, etc.? 😉